documentation/content/Analytics SEO SMO/goat-counter.md

```diff
@@ -0,0 +1,24 @@
+---
+Title: How to use Goat Counter
```

Comment:
It's actually spelled as a single word: "GoatCounter," rather than two (not that I'm overly pedantic about this, but might as well use the same 😅)